@xen-orchestra/backups/RemoteAdapter.mjs

```diff
@@ -826,6 +827,17 @@ export class RemoteAdapter {
     }
     return metadata
   }
+
+  async getTotalVmBackupSize() {
+    const vmBackups = await this.listAllVmBackups()
+    return reduce(vmBackups, (sum, backups) => sum + backups.reduce((sum, backup) => sum + backup.size, 0), 0)
+  }
+
+  // @TODO: add `getTotalXoBackupSize` and `getTotalPoolBackupSize` once `size` is implemented
+  async getTotalBackupSize() {
```

Comment:
if   `getTotalVmBackupSize`  return an object, then this method should also return a compatible object